rocky/reports/report_types/systems_report/report.py

```diff
@@ -40,6 +40,28 @@ def generate_data(self, input_ooi: str, valid_time: datetime) -> Dict[str, Any]:
 
         ip_services = {}
 
+        WEB = _("Web")
+        MAIL = _("Mail")
+        DICOM = _("Dicom")
+        DNS = _("DNS")
+        OTHER = _("Other")
```

Comment:
You might want to move these as constants out of the function, this enhances code readability a bit and also avoids unnecessary redefinition within the function